soh/soh/Enhancements/randomizer/hook_handlers.cpp

```diff
@@ -904,6 +904,10 @@ void RandomizerOnVanillaBehaviorHandler(GIVanillaBehavior id, bool* should, va_l
             *should = false;
             break;
         }
+        case VB_GIVE_ITEM_FROM_POE_COLLECTOR: {
+            *should = OTRGlobals::Instance->gRandoContext->GetItemLocation(RC_MARKET_10_BIG_POES)->IsAddedToPool();
+            break;
+        }
```

Comment:
I'm not quite as familiar with the VBifying process, but do we need a check for IS_RANDO in here? In a vanilla playthrough there's no guarantee that gRandoContext isn't a nullptr. Or at least that's the idea I had when I started down that road. It may always get instantiated right now but I wasn't necessarily intending that to be the case forever.

Reply:
No, that hook handler is only called during rando.